### ISSUE TITLE
Caching large raster operations (hitting gdalwarp system call) without filename2 creates a Frankenstein...

### ISSUE THREAD

**Issue by tati-micheletti** [PredictiveEcology/reproducible#78]
It looks like there is a problem when using cache without passing `filename2`. This appears to be happening when we hit gdalwarp (big raster) and we already have a raster with the same name saved in the cache's raster folder:
```
Copying nodata values from source  
/mnt/storage/borealBirdsAndForestry/cache/rasters/rastersrastersrastersSmallLCC2005_V1_4a_6.tif to  
 destination /mnt/storage/temp/raster/bigRasters/SmallLCC2005_V1_4a_reproj.tif.  
0...10...20...30...40...50...60...70...80...90...100 - done.  
Browse[1]> LCC05  
class       : RasterLayer  
dimensions  : 15949, 29081, 463812869  (nrow, ncol, ncell)  
resolution  : 250, 250  (x, y)  
extent      : -4260639, 3009611, -421176.8, 3566073  (xmin, xmax, ymin, ymax)  
coord. ref. : +proj=lcc +lat_1=49 +lat_2=77 +lat_0=49 +lon_0=-95 +x_0=0 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs  
data source : /mnt/storage/borealBirdsAndForestry/cache/rastersSmallLCC2005_V1_4a_reproj_2.tif  
names       : SmallLCC2005_V1_4a_reproj  
values      : 0, 255  (min, max)  
```
My guess is that when we create `SmallLCC2005_V1_4a_reproj.tif`, it is moved to destination folder, and if we have this file there, it is saved by adding a `_X`, but as it is missing a `/` after the destination path, the `raster` "folder" is getting stuck to the name of the final raster, creating this Frankenstein word AND saving in the cache folder, not the rasters folder. This should be an easy fix, though.

**Comment by achubaty** [comment]
Use of paste() instead of file.path()?

**Comment by tati-micheletti** [comment]
More probable than forgetting a `/`... hehehe

**Comment by tati-micheletti** [comment]
Fix on PR #79
